torch/_meta_registrations.py

```diff
@@ -135,42 +135,6 @@ def meta_repeat_interleave_Tensor(repeats, output_size=None):
         )
     return repeats.new_empty(output_size)
 
-@torch.library.impl(meta_lib, "_linalg_qr_helper")
```

Comment:
@Chillee we may want to keep this in Python so that we have something that works with dynamic shapes. Or maybe getting structured kernels to work in symints isn't a complete pipe dream; it actually doesn't look like it would be that difficult.